### PR TITLE
[All] Icons for Android and iOS.

### PR DESCRIPTION
**Pull request by willlarche**

### CODE REVIEW

**Comment by redbrogdon** [comment]
These LGTM, though I admit I'm no icon expert. :)

Did you generate them with [`flutter_launcher_icons`](https://pub.dartlang.org/packages/flutter_launcher_icons), or create your own from scratch?

**Comment by willlarche** [comment]
@RedBrogdon I didn't! I didn't know about that. I did it by hand.

**Review by afohrman — APPROVED** [review]
LGTM for Android

That flutter icon generator looks super cool!

File: mdc_100_series/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,8 +14,9 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="mdc_100_series"
-        android:icon="@mipmap/ic_launcher">
+        android:label="Shrine"
```
[the remark this answers is not in the document]
Comment (willlarche) [reply mdc_100_series/android/app/src/main/AndroidManifest.xml]: Tried it and it works. Done.

**Review by ianegordon — APPROVED** [review]
Looks sharp.

**Pull request merged**